packages/editor-ui/src/components/ContactPromptModal.vue

```diff
@@ -71,19 +75,30 @@ export default mixins(workflowHelpers).extend({
 	methods: {
 		closeDialog(): void {
 			if (!this.isEmailValid) {
-				this.$telemetry.track('User closed email modal', { instance_id: this.$store.getters.instanceId, email: null });
+				this.$telemetry.track('User closed email modal', {
+					instance_id: this.$store.getters.instanceId,
+					email: null,
+				});
 			}
 			this.$store.commit('ui/closeTopModal');
 		},
 		send(): void {
 			if (this.isEmailValid) {
```

Comment:
nitpick - still not sure why we need this check.. seems to me like we don't need it

Reply:
Fixed